internal/plumbing/tree_diff.go

```diff
@@ -44,6 +46,10 @@ const (
 	ConfigTreeDiffLanguages = "TreeDiff.Languages"
 	// allLanguages denotes passing all files in.
 	allLanguages = "all"
+
+	// ConfigTreeDiffFilteredRegex is the name of the configuration option
+	// (TreeDiff.Configure()) which will only compute differences based on matching this regex
```

Comment:
`which makes FileDiff consider only those files which have names matching this regexp.`